code/script.js

```diff
@@ -1,325 +1,380 @@
+    job: "lawyer",
+    accessories: ["suit"],
+    other: [],
   },
   {
-    name: 'Jude',
-    img: 'images/jude.svg',
-    hair: 'black',
-    eyes: 'green',
-    accessories: [],
-    other: []
+    name: "Wendy",
+    img: "images/Wendy.png",
+    color: "black",
+    job: "mooch",
+    accessories: ["jewelry"],
+    other: ["royal"],
   },
   {
-    name: 'Julie',
-    img: 'images/julie.svg',
-    hair: 'black',
-    eyes: 'brown',
-    accessories: ['glasses', 'hat'],
-    other: []
+    name: "Yeti",
+    img: "images/Yeti.png",
+    color: "black",
+    job: "designer",
+    accessories: ["hat", "glasses"],
+    other: [],
   },
-]
+];
 
-// Global variables
-let secret
-let currentQuestion
-let charactersInPlay
+let secret; // will be the secret character
+let currentQuestion; // will be the current question object
+let charactersInPlay; //will be an array of all characters left in the game
+let questions; //Question counter
 
-// Draw the game board
+//Generates the board with all the dogs. Starts with reset to clear dogs if filter is applied.
 const generateBoard = () => {
-  board.innerHTML = ''
-  charactersInPlay.forEach((person) => {
+  board.innerHTML = "";
+  charactersInPlay.forEach((dog) => {
     board.innerHTML += `
-      <div class="card">
-        <p>${person.name}</p>
-        <img src=${person.img} alt=${person.name}>
-        <div class="guess">
-          <span>Guess on ${person.name}?</span>
-          <button class="filled-button small" onclick="guess('${person.name}')">Guess</button>
+        <div class="card">
+            <img src=${dog.img} alt=${dog.name}/>
+            <div class="guess">
+            <span>Guess on ${dog.name}?</span>
+            <button class="filled-button small" onclick="guess('${dog.name}')">Guess</button>
+            </div>
+            <p>${dog.name}</p>
         </div>
-      </div>
-    `
-  })
-}
+        `;
+  });
+};
 
-// Randomly select a person from the characters array and set as the value of the variable called secret
+//Generating secret character
 const setSecret = () => {
-  secret = charactersInPlay[Math.floor(Math.random() * charactersInPlay.length)]
-}
+  secret =
+    charactersInPlay[Math.floor(Math.random() * charactersInPlay.length)];
+};
 
-// This function to start (and restart) the game
+//Starting function
 const start = () => {
-  // Here we're setting charactersInPlay array to be all the characters to start with
-  charactersInPlay = CHARACTERS
-  // What else should happen when we start the game?
-}
+  charactersInPlay = CHARACTERS; //everyone is in play at the beginning
+  setSecret(); //Set secret character
+  generateBoard(); //Draws the board
+  winOrLose.style.display = "none"; //needs to be cleared if someone has restarted
+  board.style.display = "flex"; //board needs to be visible if "play again" was clicked
+  questions = 5; //Number of guesses at start
+  questionCounter.innerHTML = `5`; //Number of guesses shown to player
+};
 
```

Comment:
The number of guesses is a very fun feature that adds to how playable and challenging the game is. The start function displays clarity in coding. Perhaps the number of guesses can be not hard-coded and instead as:
`questionCounter.innerHTML = `${questions}`;` in the case of wanting to change the game's rules later.